src/action-handler.ts

```diff
@@ -220,3 +224,66 @@ export const actionHandlerBind = (element: ActionHandlerElement, options: Action
 export const actionHandler = directive((options: ActionHandlerOptions = {}) => (part: PropertyPart): void => {
   actionHandlerBind(part.committer.element as ActionHandlerElement, options);
 });
+
+export const executeAction = (
```

Comment:
Sadly i could not use any helpers from `custom-card-helpers` in place of this :disappointed: 
They specifically check if the action is "tap" and if so take the value "tap_action"
Thus i'd be unable to add a pressed or released action

Reply:
Ideally for this I'd PR something there. That's how we dealt with it to keep all the features available for everyone in the past.

Reply:
I'll definitely give it a shot, i've got a bit of an idea to implement it in a (hopefully) nice way there

Reply:
The idea i had worked, simply keep the abstraction with "action" in the HandleAction function and move the actual logic to a new HandleActionConfig function https://github.com/custom-cards/custom-card-helpers/pull/20/files#diff-c30c4816c042b6020567e4f025b9ab42

Now we'll just have to wait until it's merged. I have also temporarily added the file to this PR so the code will keep functioning. But will remove it once the PR in card helpers has been merged :smiley:

Reply:
Totally missed the cch PR, sorry about that. It has been merged and is in release https://www.npmjs.com/package/custom-card-helpers/v/1.6.8

Reply:
Any news on this? :)

Reply:
Yeah! I could also use this functionality. Call a service on press, and call another service on release.
With this, you could create things like remote controls, where as long as you keep pressing, the roller-blinds go down. That kind of stuff. Please review and merge :)